app-modules/knowledge-base/src/Filament/Resources/KnowledgeBaseItemResource/Pages/ListKnowledgeBaseItems.php

```diff
@@ -114,6 +131,34 @@ public function table(Table $table): Table
                     ->multiple()
                     ->preload(),
                 TernaryFilter::make('public'),
+                Filter::make('created_at')
+                    ->label('Created After')
+                    ->form([
+                        DatePicker::make('created_after')
+                            ->native(false)
+                            ->format('d-m-Y'),
+                    ])
+                    ->query(function (Builder $query, array $data): Builder {
+                        return $query
+                            ->when(
+                                $data['created_after'],
+                                fn (Builder $query, $date): Builder => $query->whereDate('created_at', '>=', $date),
+                            );
+                    }),
+                Filter::make('updated_at')
```

Comment:
Let's add a test to `app-modules/knowledge-base/tests/KnowledgeBaseArticle/ListKnowledgeBaseArticlesTest.php` for each of these filters.

Creating KB articles, triggering the filter, then confirming that you do see that ones you would expect and not the ones that you would expect to not be listed.